Document/0x06b-Basic-Security-Testing.md

```diff
@@ -359,7 +351,29 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
 
 The following sections describes on how to retrieve basic information of an iOS app, that might be useful during a penetration test.
 
--- ToDo Objection: https://github.com/OWASP/owasp-mstg/issues/1247
+Let's focus first on objection. In the following example Frida is running as server on a jailbroken device and the app iGoat is running in the foreground. To attach to a process in this scenario you need to use the flag `--gadget` with the process name. When using objection you can retrieve different kind of information, where `env` will show you all the directory information of iGoat.
+
+```bash
+$ objection  --gadget "iGoat-Swift" explore
```

Comment:
maybe first explain how to patch the ipa, get your codesign, etc. ? Or is that done somewhere else? If so: maybe you can refer to that, or start by explaining how that is done.

Reply:
That's not needed in this case, as the frida server is running on iOS. The IPA was not patched. 

But I added another sentence to clarify also on what to do on a non-jb device with a link to objection wiki